br.unb.cic.mcsl/src/br/unb/cic/mcsl/MetaCrySL.xtext

```diff
@@ -215,8 +215,9 @@ ArithmeticExp
 AtomicConstraintExp 
  : {Value} val = Value
  | {Variable} var = ID
+ | {ElementsCall} el ="elements(" var = ID ")" 
  | '(' ConstraintExp ')'
- | {FunctionCall} methodName=ID '(' params += Value (',' params += Value)* ')' 
```

Comment:
@faustocarva considering methodName = ID, I think that it is more generic than these options ("alg" | "mode" | ..). What do you think? I mean, I am wondering it is possible to represent all, mode, and so on using methodName = ID.

Reply:
Yes, it is. But it might generates incorrect or malformed crysl products from metacrysl files. So the derivation process could be not "sound" in the face of a correct crysl file. That's the reason why  this PR was created, to get metacrysl closer to a correct crysl file.